README.md

```diff
@@ -1,11 +1,9 @@
 ## Install
 
 ```bash
-npm install eslint
-npm install eslint-plugin-promise eslint-plugin-standard eslint-plugin-react
-npm install eslint-config-standard
-npm install eslint-config-punkave
+npm install eslint eslint-plugin-promise eslint-plugin-standard eslint-plugin-react eslint-config-standard eslint-config-punkave --save-dev
```

Comment:
Done. See 55e68851e58757a2002efe2280aac3cdaf023407 and 6d7bae55da0e7e0a57bc591090dbd57a303a4ece